transports/janus_http.c

```diff
@@ -1062,6 +1064,11 @@ void janus_http_session_over(janus_transport_session *transport, guint64 session
 	janus_mutex_unlock(&sessions_mutex);
 }
 
+void janus_http_session_claimed(janus_transport_session *transport, guint64 session_id) {
+	/* Session has been claimed -- is there anything to do here? */
+	JANUS_LOG(LOG_WARN, "Session has been claimed: (%"SCNu64"), has been claimed\n", session_id);
```

Comment:
This is where I'd update the `sessions` hashmap, for instance: whether the session existed in the map or not, a reclaim here would tell the transport "look, this session exists, and it's handled by this new transport". If it existed, it would sever the relationship between the old transport and the session ID; in both cases, long poll requests for the sessions wouldn't fail, as we would have a mapping there.